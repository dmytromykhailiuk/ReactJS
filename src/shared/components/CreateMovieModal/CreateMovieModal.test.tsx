import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import { CreateMovieModal } from '.';

describe('CreateMovieModal', () => {
  it('should match first snepshot', () => {
    const wrapper = shallow(
      <CreateMovieModal movie={{} as any} onCloseModal={() => {}} onCloseWithSaving={() => {}} />,
    );
    expect(toJson(wrapper)).toMatchSnapshot('first');
  });
});
