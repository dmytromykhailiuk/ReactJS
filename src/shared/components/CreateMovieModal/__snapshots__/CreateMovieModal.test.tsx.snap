// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateMovieModal should match first snepshot: first 1`] = `
<Component
  header="ADD MOVIE"
  onCloseModal={[Function]}
>
  <MovieForm
    onSubmitForm={[Function]}
    submitButtonLabel="SUBMIT"
  />
</Component>
`;
